### PR TITLE
Update timeline for BIL/LIL retirement, change some wording

### PR DESCRIPTION
**Pull request by wrall**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ If the migration fails with a grade of **Poor**, you cannot force the migration
...
+Through the end of 2020, the [BadItemLimit and LargeItemLimit parameters](https://docs.microsoft.com/powershell/module/exchange/new-moverequest) will remain available as options. You can specify a value for the *BadItemLimit* and *LargeItemLimit* parameters when using cmdlets or you can fill in the BadItemLimit or LargeItemLimit box in the EAC UI. When you specify a BadItemLimit or LargeItemLimit, the old migration method is used and the DataConsistencyScore is not calculated.
 
-If neither the *BadItemLimit* parameter nor the *LargeItemLimit* parameter are specified, or if the boxes in the Exchange admin center wizard are left blank, then the new migration method and DataConsistencyScore are used.
+If neither the *BadItemLimit* parameter nor the *LargeItemLimit* parameter are specified, or if the boxes in the classic Exchange admin center wizard are left blank, then the new migration method and DataConsistencyScore are used.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/track-prevent-data-loss-dcs.md:68]:
```suggestion
If neither the *BadItemLimit* parameter nor the *LargeItemLimit* parameter is specified, or if the boxes in the classic Exchange admin center wizard are left blank, then the new migration method and DataConsistencyScore are used.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**